rust/xaynet-analytics/src/database/common.rs

```diff
@@ -172,7 +132,7 @@ impl TryFrom<&str> for RelationalField {
 
 impl Into<String> for RelationalField {
     fn into(self) -> String {
-        format!("{:?}={:?}", self.value, self.collection_name)
+        [self.value, self.collection_name].join("=")
```

Comment:
then in `TryFrom<&str>` above the data str contained some extra formatting `"\"` characters and i didn't find a clean way to strip them, while joining the array just worked

Reply:
I don't know why he choose to change but with `join` is more direct, and a bit more efficient since we don't have to parse the string and call into the `Debug` trait.